services/alertmanager/alertmanager.go

```diff
@@ -107,10 +110,8 @@ func (svc *Service) createDataDir() {
 	}
 }
 
-// generateBaseConfig generates /srv/alertmanager/alertmanager.base.yml if it is not present.
-//
-// TODO That's a temporary measure until we start generating /etc/alertmanager.yml
-// using /srv/alertmanager/alertmanager.base.yml as a base. See supervisord config.
```

Comment:
We should update supervisord config to use /etc/alertmanager.yml instead of /srv/alertmanager/alertmanager.base.yml

Reply:
updated https://github.com/percona/pmm-managed/pull/514/commits/d9edf32bcd500a85a8769da524bdad306447d325